warehouse/query-core/src/main/java/datawave/query/jexl/nodes/IndexHoleMarkerJexlNode.java

```diff
@@ -29,7 +28,7 @@ public static String label() {
      * @return the new marker node
      * @see QueryPropertyMarker#create(JexlNode, Function)
      */
-    public static IndexHoleMarkerJexlNode create(JexlNode node) {
+    public static JexlNode create(JexlNode node) {
```

Comment:
This can return IndexHoleMarkerJexlNode if we return a MARKER from `create(source, constructor)`.